ecs/templates/npm.json.tpl

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
Is it worth combining the `npm.json.tpl` and the `plugin_updates.json.tpl` to a single policy and use variables? They look identical to me?

Reply:
Good idea, thanks They are identical apart from the name so I've added a parameter in for that. This will help if we need to do this with any images in the future as we can reuse this again.